back/engines/commercial/multi_tenancy/app/models/multi_tenancy/patches/user_confirmation/user.rb

```diff
@@ -1,15 +0,0 @@
-# frozen_string_literal: true
-
-module MultiTenancy
-  module Patches
-    module UserConfirmation
-      module User
-        private
-
-        def use_fake_code?
-          super || AppConfiguration.instance.host == 'e2e.stg.citizenlab.co'
```

Comment:
Now we have got rid of this - what happens in the e2e tests - does Rails.env.development get set to true there?

Reply:
We haven't used `e2e.stg.citizenlab.co` for e2e tests for 2 years. 

Currently, a docker compose server for them is indeed started in development environment https://github.com/CitizenLabDotCo/citizenlab/blob/3de680a802fed693aa2b66d9b593147941d4f118/.circleci/config.yml#L627